server/tests/end_to_end_decentralized.ts

```diff
@@ -166,10 +166,10 @@ describe('end to end', function () {
   }
 /* eslint-disable */
   async function testWeightSharingSecure (): Promise<Weights> {
-    // expected --> [6, 7, 13]
-    makeClientSecure([3, 3, 3])
-    makeClientSecure([4, 5, 6])
-    return await makeClientSecure([11, 13, 30])
+    // expected --> [.002, 7, 27]
```

Comment:
add some comment here if the array is over model dimension = 3 or number of clients =3

Reply:
I don't understand-- add a comment saying what about the model dimension or client numbers? currently the model doesn't undergo any training in this test -- it is just testing the communication of updated weights between clients.